baseplate/events/queue.py

```diff
@@ -170,6 +170,11 @@ def serialize_v2_event(event):
     return Serializer.serialize(_V2_PROTOCOL_FACTORY, event)
 
 
+class EventLogger(object):
+    def log(self, **kwargs):
+        pass
```

Comment:
Maybe this should be called `EventLoggerInterface` and `raise NotImplementedError` rather than just `pass`ing.

Reply:
I _think_ we want `event_name` or just `name` to be an argument so you don't have to define a separate logger object for each event you want to send.

Reply:
I think it's fine for the ABC to just be EventLogger; it's functionally an interface but we don't put that word in the names in other places in baseplate.

Reply:
We won't need to define a separate logger for each event type; we can create a logger which is relatively dynamic, accepting "arbitrary" params to construct an event. That also doesn't preclude us from creating a logger which *expects* an event_type parameter to be passed along.
I think you'll see what I mean in an upcoming PR (feel free to block this one in the interim if you're really concerned)